.github/workflows/uffizzi-build.yml

```diff
@@ -55,6 +55,11 @@ jobs:
           images: registry.uffizzi.com/${{ env.UUID_TAG_APP }}
           tags: type=raw,value=60d
 
+      - name: Add examples directory to Dockerfile
+        run: |
+          echo "COPY examples /app/examples" >> ./example-app/packages/backend/Dockerfile
+          cat ./example-app/packages/backend/Dockerfile
```

Comment:
just for debug?

Reply:
For now yes, will remove when it's all working. Looks like the container is not being deployed properly :shrug: